src/RoleObjects/FormData.test.js

```diff
@@ -0,0 +1,160 @@
+import * as createjs from 'createjs-module';
+import AccessibilityModule from '../index';
+
+describe('FormData', () => {
+  describe('register role', () => {
+    let canvasEl;
+    let parentEl;
+    let stage;
+    let container;
+    let cjsForm;
+    let formEl;
+    let actionVal;
+    let shouldAutoComplete;
+    let charSetVal;
+    let enctypeVal;
+    let methodVal;
+    let nameVal;
+    let targetVal;
+
+    beforeEach(() => {
+      canvasEl = document.createElement('canvas');
+      parentEl = document.createElement('div');
+      stage = new createjs.Stage(canvasEl);
+      container = new createjs.Container();
+      cjsForm = new createjs.Shape(); // dummy object
+      actionVal = 'formsubmit.php';
+      shouldAutoComplete = false;
+      charSetVal = 'utf-8';
+      enctypeVal = 'text/plain';
+      methodVal = 'get';
+      nameVal = 'form_name';
+      targetVal = '_blank';
+
+      AccessibilityModule.register({
+        displayObject: container,
+        role: AccessibilityModule.ROLES.MAIN,
+      });
+      AccessibilityModule.setupStage(stage, parentEl);
+      stage.accessibilityTranslator.root = container;
+      stage.addChild(container);
+
+      AccessibilityModule.register({
+        accessibleOptions: {
+          action: actionVal,
+          autoComplete: shouldAutoComplete,
+          charSet: charSetVal,
+          enctype: enctypeVal,
```

Comment:
- interesting problem here (and this cropped up a few times in the work for this PR):
```Warning: Invalid DOM property `enctype`. Did you mean `encType`?```
I'm reticent to change the API to match what's expected, I'd rather go the route of fixing the internals and _deprecating_ the methods that would go away. Thoughts?

Reply:
I agree with deprecating the existing method and standardizing on the actual DOM accessor names (probably a good follow-on PR, though, IMO—unless you feel it's in scope).

Reply:
good question (now vs follow-up), it's not explicitly called for in the current task but I feel like if nothing else the internals need to get fixed (because that's essentially broken code/tech debt)

maybe deprecation is the follow-on